### DISCUSSION TITLE
Microsoft shoule enable from Windows command prompt directly run Linux command and reverse versa

### DISCUSSION THREAD

**Discussion by wzis** [community/community#20258] [Other Feature Feedback, Questions, & Ideas]
And let root account of the WSL be able to search and read all Windows dirs/files; plus make Windows commands able to be moved to any dir to run properly, that way, but using the Linux security software, Windows will be more secure.

**Comment by hashfx** [comment]
Type ```wsl -d <DistributionName>``` in Command Prompt or Windows Terminal and change <DistributionName> with your Distro name
For name of your distro, type:
```wsl.exe --list --all```

For Example:
To open Kali Linux Terminal on Windows Terminal:
```
wsl -d kali-linux
```

That's how it should work:
```js
C:\Users\Dragon>wsl -d kali-linux  // type command
// it would open Linux Terminal in Windows Terminal
┌──(hashfx㉿Dragon)-[/mnt/c/Users/Dragon]
└─$
```

**Reply by wzis** [reply]
What I mean to directly run Linux command in Windows command prompt and directly run Windows command in Linux tty is when you have the WSL installed, no matter in which env (Windows PowerShell/command prompt or Linux TTY), you can directly run a command without typing extra things, for example in Windows command prompt, we should be able to run "df -h" or "ls -ltR", and in Linux TTY, just run "notepad" it should work.

**Comment by Orzechowski-gif** [comment]
Kamil Orzechowski  historia
